src/components/ContactDetails.vue

```diff
@@ -570,7 +625,22 @@ export default {
 
 			return ''
 		},
+		profilePageLink() {
+			return generateUrl('/u/{userId}', { userId: getCurrentUser().uid })
 
+		},
+		phoneNumberProperty() {
+			return this.localContact.properties.find(property => property.name === 'tel')
+		},
+		emailAddressProperty() {
+			return this.localContact.properties.find(property => property.name === 'email')
+		},
+		emailAddress() {
+			return this.emailAddressProperty.getFirstValue()
+		},
+		phoneNumber() {
+			return this.phoneNumberProperty.getFirstValue()
```

Comment:
@kesselb we had the solution right in front our eyes. getFirstValue was what we needed and not localvalue and some property value. We focused too much on the specific properties :)